app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/LinkPreviewViewHolder.java

```diff
@@ -378,4 +330,62 @@ public void onFailure(Call<SkillRatingResponse> call, Throwable t) {
 
         });
     }
+
+    private void setResponseListener(final Context currContext) {
```

Comment:
Remove context as parameter and return the listener rather than having void function